pkgs/top-level/all-packages.nix

```diff
@@ -15739,7 +15739,10 @@ with pkgs;
   mitama-cpp-result = callPackage ../development/libraries/mitama-cpp-result { };
 
   mitscheme = callPackage ../development/compilers/mit-scheme
-    { texLive = texlive.combine { inherit (texlive) scheme-small epsf texinfo; }; };
+    {
+      texLive = texlive.combine { inherit (texlive) scheme-small epsf texinfo; };
+      texinfo = texinfo6;
```

Comment:
Should we use a different version of texinfo in texlive?

Reply:
It looks like texlive has only one version of texinfo: https://github.com/nixos/nixpkgs/blob/edefe53eaaa277c8b5f9598b912f13690f7274de/pkgs/tools/typesetting/tex/texlive/tlpdb.nix#L36105-L36111